coffee_lib/src/url.rs

```diff
@@ -18,7 +18,7 @@ pub struct URL {
 }
 
 /// Handle GitHub HTTP links
-fn remove_dot_git_from_url(url: &str) -> &str {
+pub(crate) fn remove_dot_git_from_url(url: &str) -> &str {
```

Comment:
Why do we need to add `pub(crate)`?
I don't see it used anywhere other than this file

Reply:
it is used inside the tests, and the test is another module